### PR TITLE
Initial checkin of rna-collect-multiple-metrics workflow

### LINKED ISSUE #17: Add collect-multiple-metrics workflow
For AMP PD RNA data, a collect-multiple-metrics workflow was run.

### PR DESCRIPTION
**Pull request by wnojopra**
This PR includes the rna-collect-multiple-metrics workflow which executes picard CollectInsertSizeMetrics, CollectAlignmentSummaryMetrics, QualityScoreDistribution, and MeanQualityByCycle on a BAM file.

Fixes #17 .

### CODE REVIEW

File: rna/rna-collect-multiple-metrics/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  - VM disk size
+  - Num CPU cores
+  - VM memory
+  - Runtime zones, ex: "us-central1-a us-central1-b"
+  - Number of times to try the workflow with a preemptible VM before
+    falling back to a full-price VM.
+
+Outputs:
+- Per-sample:
+  - File outputFiles - glob(*)
```
Comment (wnojopra) [inline rna/rna-collect-multiple-metrics/README.md]: Seems like the asterisk here is making github display the rest of the file in italics. I think you can safely ignore the fact that its displayed here in italics.

File: rna/rna-collect-multiple-metrics/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# RNACollectMultipleMetrics with Picard
+
+This workflow runs `picard CollectInsertSizeMetrics`, `picard CollectAlignmentSummaryMetrics`, `picard QualityScoreDistribution`, and `picard MeanQualityByCycle` on a BAM file
```
Comment (mbookman) [inline rna/rna-collect-multiple-metrics/README.md]: This workflow runs the following `picard` commands on a BAM file:

- [CollectInsertSizeMetrics](https://software.broadinstitute.org/gatk/documentation/tooldocs/4.0.1.2/picard_analysis_CollectInsertSizeMetrics.php)
- [CollectAlignmentSummaryMetrics](https://software.broadinstitute.org/gatk/documentation/tooldocs/4.0.1.2/picard_analysis_CollectAlignmentSummaryMetrics.php)
- [QualityScoreDistribution](https://software.broadinstitute.org/gatk/documentation/tooldocs/4.0.1.2/picard_analysis_QualityScoreDistribution.php)
- [MeanQualityByCycle](https://software.broadinstitute.org/gatk/documentation/tooldocs/4.0.1.2/picard_analysis_MeanQualityByCycle.php)

File: rna/rna-collect-multiple-metrics/README.md (hunk 1 above)
Comment (mbookman) [reply rna/rna-collect-multiple-metrics/README.md]: Let's list all of the output files:

- &lt;sample-id&gt;.Multiple_Metrics.alignment_summary_metrics
- &lt;sample-id&gt;.Multiple_Metrics.base_distribution_by_cycle.pdf
- &lt;sample-id&gt;.Multiple_Metrics.base_distribution_by_cycle_metrics
- &lt;sample-id&gt;.Multiple_Metrics.insert_size_histogram.pdf
- &lt;sample-id&gt;.Multiple_Metrics.insert_size_metrics
- &lt;sample-id&gt;.Multiple_Metrics.quality_by_cycle.pdf
- &lt;sample-id&gt;.Multiple_Metrics.quality_by_cycle_metrics
- &lt;sample-id&gt;.Multiple_Metrics.quality_distribution.pdf
- &lt;sample-id&gt;.Multiple_Metrics.quality_distribution_metrics

File: rna/rna-collect-multiple-metrics/rna-collect-multiple-metrics.inputs.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  "##_COMMENT1": "Set your per-sample values: name and BAM path, and intervals file path.",
+  "RNACollectMultipleMetrics.sample_name": "__INSERT_SAMPLE_NAME__",
+  "RNACollectMultipleMetrics.bam_file": "__INSERT_GCS_PATH_TO_BAM_FILE__",
+
+  "##_COMMENT2": "Path to reference FASTA file.",
+  "RNACollectMultipleMetrics.reference_sequence_file": "__INSERT_GCS_PATH_TO_REF_FASTA_FILE__",
```
Comment (mbookman) [inline rna/rna-collect-multiple-metrics/rna-collect-multiple-metrics.inputs.json]: For the other workflows in this repo, we included the path to the reference file. For this one it should be:

```
gs://amp-pd-transcriptomics/inputs/reference/gencode/GRCh38.primary_assembly.genome.fa.gz
```

File: rna/rna-collect-multiple-metrics/rna-collect-multiple-metrics.wdl [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+##   - VM disk size
+##   - Num CPU cores
+##   - VM memory
+##   - Runtime zones, ex: "us-central1-a us-central1-b"
+##   - Number of times to try the workflow with a preemptible VM before
+##     falling back to a full-price VM.
+##
+## Outputs:
+## - Per-sample:
+##   - File outputFiles - glob(*)
```
Comment (mbookman) [inline rna/rna-collect-multiple-metrics/rna-collect-multiple-metrics.wdl]: Let's list the outputs.

File: rna/rna-collect-multiple-metrics/rna-collect-multiple-metrics.wdl [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+    memory: vm_memory
+    cpu: num_cpu_cores
+    preemptible: preemptible_tries
+    zones: runtime_zones
+  }
+  output {
+    Array[File] metrics_output_path = glob("${sample_name}.Multiple_Metrics.*")
+  }
+}
```
Comment (mbookman) [inline rna/rna-collect-multiple-metrics/rna-collect-multiple-metrics.wdl]: nit: add a newline at the end of the file

**Review by mbookman — CHANGES_REQUESTED** [review]

**Review by mbookman — APPROVED** [review]
LGTM

**Review by ha-duong-technome — APPROVED** [review]

**Pull request merged**